### PR TITLE
feature/TC-03 Styled System and Styled Component

### PR DESCRIPTION
**Pull request by hqadeer-nisum-com**
- Create an example pages using with style component.
- Create an style system, create example theme which is using in the components.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,11 +8,14 @@
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.5.2",
     "@types/node": "^16.11.48",
-    "@types/react": "^18.0.17",
-    "@types/react-dom": "^18.0.6",
```
Comment (hiqbalmallick) [inline package.json:12]: why these two dev dependencies are deleted?

Comment (hqadeer-nisum-com) [reply package.json:12]: because its style component so thats why i delete them and add them into routing pr.

Comment (hiqbalmallick) [reply package.json:12]: they come with the default installation of react app using typescript, plus the code is moved below with a separate dev-dependencies block if you can see.

File: src/example/pages/About.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const AboutPage: React.FunctionComponent<IAboutPageProps> = (props) => {
+  return (
+    <div>
+      <p>about page</p>
+      <div>
+        <div>
+          <Button>Back To Home</Button>
+        </div>
+      </div>
+    </div>
```
Comment (hiqbalmallick) [inline src/example/pages/About.tsx]: div inside empty div?

Comment (hqadeer-nisum-com) [reply src/example/pages/About.tsx]: i will change it shortly.

File: package.json (hunk 1 above)
Comment (hqadeer-nisum-com) [reply package.json:12]: oh yeah yeah hamza bhai i saw that in my code these dependencies added in my package.json file. but these dependencies added twisty by mistake so thats why i did remove that, do i add them again ?

Comment (hiqbalmallick) [reply package.json:12]: move them up here again from below.

Comment (hqadeer-nisum-com) [reply package.json:12]: resolved hamza bhai.

File: src/App.tsx [hunk 3]
```diff
@@ -2,15 +2,19 @@ import "./App.css";
...
 
 function App() {
   return (
     <Provider store={store}>
-      <div className="App">
-        <Application />
-        <Example />
-      </div>
+      <ThemeProvider theme={theme}>
+        <GlobalStyle />
```
Comment (hiqbalmallick) [inline src/App.tsx]: Give this a meaningful name, like main layout or something, plus does <Example /> component should also be a part of global theme right?

Comment (hqadeer-nisum-com) [reply src/App.tsx]: yes hamza bhai, its a part of our components.

**Review by hiqbalmallick — APPROVED** [review]

**Pull request merged**